airbyte-webapp/src/components/ui/Card/Card.tsx

```diff
@@ -2,11 +2,14 @@ import classNames from "classnames";
 import React from "react";
 
 import { H5 } from "components/base/Titles";
+import { Text } from "components/ui/Text";
 
+import { InfoTooltip } from "../Tooltip";
 import styles from "./Card.module.scss";
 
 export interface CardProps {
   title?: React.ReactNode;
+  description?: React.ReactNode;
```

Comment:
Given that this is a basic UI component, I'm not sure we want to add something that specific as the description which renders as an Info icon here, or rather want to stay more generic, and e.g. just allow passing in `headerComponents` that would render there. @edmundito what are your thoughts around this?